### ISSUE TITLE
Deliver basic IDML-TEI transformation for TEI stylesheets

### ISSUE THREAD

**Issue by gerritbruening** [faustedition/faust-gen-html#596]
When I told @raffazizzi that we went from TEI to IDML in our project he asked whether a basic versions of these pipeline could be added to the TEI stylesheets. What do you think about this, @pglatza?
(not really a org:faustedition issue)

**Comment by pglatza** [comment]
Into the repo https://github.com/raffazizzi/Stylesheets ?

Yes, I would agree, to sum up.

Unfortunately, the TEI to IDML conversion for faustedition has a lot of customizations for the faust project builtin. Not yet separated from the 'vanilla' part of tei2idml. I do not have the time to provide a basic version, at the moment.
Furthermore, the conversion needs some infrastructure of our transpect.io framework, not just some `xsl` and some `xpl` files.
Maybe we can provide a `README` in an `idml` sub directory of the mentioned repository, after the basic version is created.

**Comment by gerritbruening** [comment]
See also http://www.numedif.fr/metopes.html (Alexei Lavrentiev is presenting about this at TEI 2018).

**Comment by pglatza** [comment]
When playing with xml and InDesign, you get faster results when relying on `aid` attributes. But there are many limitations (for example no indexterms, longer waiting time for import, maybe 'out of memory' errors for some projects like dictionaries, a semi-optimal editable xml structure; object styles?).
At the beginning (I think it was in 2009) we also started with aid attributes. But we had to switch to our idml synthesis we are still using.
